### DISCUSSION TITLE
How do I test the Card in local running in localhost:8080?

### DISCUSSION THREAD

**Discussion by JenuelDev** [jofftiquez/vue-stripe-checkout#290] [Q&A] [answered]
How do I test the Card in local running in localhost:8080?

I am getting this. "Vue Stripe will not work on an insecure host. Make sure that your site is using TCP/SSL"

![image](https://user-images.githubusercontent.com/31676163/178626943-4499a0fa-f2b2-40bb-9c8f-148168edd98c.png)

**Comment by jofftiquez** [comment]
Hi I temporarily disabled this warning in [v4.4.4](https://github.com/vue-stripe/vue-stripe/releases/tag/v4.4.4). Thank you.

**Comment by JenuelDev** [comment, marked as answer]
Sorry, I forgot to update something here... it seems when using in local machine or for testing pusposes. It seems it works when adding a directive if `:testMode` to true.
